### PR TITLE
PSEC-2027

### PR DESCRIPTION
**Pull request by baarkerlounger**
Run script from lambda with stored creds rather than locally/with .env creds

### CODE REVIEW

File: bitwarden_manager/temp/role_updater.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # To be removed once run in production
 
 import logging
-import os
 from typing import Any, Dict, List
 from requests import HTTPError, get
-from bitwarden_manager.clients.bitwarden_public_api import BitwardenPublicApi, session, UserType
+from bitwarden_manager.clients.bitwarden_public_api import session, UserType
```
Comment (wunzeco) [inline bitwarden_manager/temp/role_updater.py:6]: Does this `session` attribute of `bitwarden_manager.clients.bitwarden_public_api` make an authenticated session available for further calls in this module?

File: bitwarden_manager/temp/role_updater.py [hunk 2]
```diff
@@ -107,10 +71,6 @@ def get_team_admin_users(self, teams: List[str]) -> List[str]:
 
 
 class BitwardenApi:
-    def __init__(self) -> None:
-        self.config = Config()
-        self.bitwarden_api = self.config.get_bitwarden_public_api()
```
Comment (wunzeco) [inline bitwarden_manager/temp/role_updater.py:112]: I wonder if the `session` attribute of `bitwarden_manager.clients.bitwarden_public_api` makes an authenticated session available for reuse in this class?

File: bitwarden_manager/temp/role_updater.py (hunk 1 above)
Comment (baarkerlounger) [reply bitwarden_manager/temp/role_updater.py]: Yes for as long as the token is valid https://github.com/hmrc/bitwarden-manager/blob/main/bitwarden_manager/clients/bitwarden_public_api.py#L108

Comment (baarkerlounger) [reply bitwarden_manager/temp/role_updater.py]: Good point re-not having called __fetch_token. Updated.

**Review by wunzeco — APPROVED** [review]

**Pull request merged**